### DISCUSSION TITLE
Welcome to lazypredict Discussions!

### DISCUSSION THREAD

**Discussion by shankarpandala** [shankarpandala/lazypredict#274] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kermezliikram** [comment]
Hello ! 
I'm using your package for a project in my work, thank you very much for Lazypredict !
I have a question about sklearn.Pipeline please, I want to get the coefficients (intercept...) for exemple like y = a*x+b, i need the a and b coefficient from sklearn.Pipeline, I searched in sklearn documentation but I didn't find what I wanted.
Do you have any idea about how to do it please ? 
Ikram.

**Reply by shankarpandala** [reply]
Every sklean model gives the parameters and its values. The same applies to pipelines also. You can get the coefficients.

**Comment by BradKML** [comment]
Hope that you can see this and bring in new collabs https://github.com/shankarpandala/lazypredict/discussions/443
